go.mod

```diff
@@ -4,7 +4,7 @@ go 1.13
 
 require (
 	github.com/jteeuwen/go-bindata v3.0.8-0.20151023091102-a0ff2567cfb7+incompatible
-	github.com/open-cluster-management/api v0.0.0-20200506150956-355c9d6ef16b
+	github.com/open-cluster-management/api v0.0.0-20200512021938-5f0d5c12ea67
```

Comment:
I wonder if we should start tagging releases soon, I have an allergic reaction to go mod pseudoversions